src/OrbitBase/include/OrbitBase/Typedef.h

```diff
@@ -190,9 +186,9 @@ auto LiftAndApply(Action&& action, Arg&& arg, Args&&... args) {
 }
 
 template <typename TypedefType>
-inline constexpr bool kHasZeroMemoryOverheadV =
-    sizeof(TypedefType) == sizeof(typename TypedefType::Value) &&
-    std::alignment_of_v<TypedefType> == std::alignment_of_v<typename TypedefType::Value>;
+inline constexpr bool kHasZeroMemoryOverheadV = true;
+// sizeof(TypedefType) == sizeof(typename TypedefType::Value) &&
+// std::alignment_of_v<TypedefType> == std::alignment_of_v<typename TypedefType::Value>;
```

Comment:
I guess this slipped into the PR unintentionally? :-)

Reply:
Sure.